property-service/src/main/java/org/egov/ps/service/EnrichmentService.java

```diff
@@ -273,6 +278,45 @@ public void enrichCreateApplication(ApplicationRequest request) {
 		}
 	}
 
+	public void enrichProcessNotifications(ApplicationRequest request) {
+		// TODO Auto-generated method stub
+		// call processNotification method
+
+		request.getApplications().forEach(application -> {
+
+			List<Map<String, Object>> notificationConfigs = mdmsservice.getNotificationConfig("notifications", request.getRequestInfo(), application.getTenantId(), application);
+			processNotification(notificationConfigs, application);
+
+		});
+
+	}
+
+	public void processNotification(List<Map<String, Object>> notificationConfigs, Application enrichedApplication) {
+		if (!CollectionUtils.isEmpty(notificationConfigs)) {
+			ObjectMapper mapper = new ObjectMapper();
+			List<Notifications> notificationList = mapper.convertValue(notificationConfigs, new TypeReference<List<Notifications>>() { });
+
+			notificationList.forEach(data -> {
+				data.setContent(data.getContent()
+						//app.getModuleType() == null ? "" : app.getModuleType()
+						.replace("{createdBy.name}",enrichedApplication.getAuditDetails() == null ? "" : enrichedApplication.getAuditDetails().getCreatedBy())
```

Comment:
These should be replaced dynamically as they are valid JsonPath.

Reply:
@rajiv-transerve  I checked but from mdmsservice.getNotificationConfig method will give me "List<Map<String, Object>> notificationConfigs" list so i used this way code ... i checked yesterday but not getting ...its good @rajiv-transerve  we discuss again